src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -417,6 +418,11 @@
     )
     private String respondentUserId;
 
+    @CCD(
+        label = "Link to online petition",
+        access = {DefaultAccess.class}
+    )
+    private Document minipetitionlink;
```

Comment:
I think it should be called mini application ?

Also do we need to store the link in case data ?

Reply:
This is set so that it is available in page configuration as below. Would there be any other way getting reference to it? 
  .pageLabel("Review the petitioner's application")
            .readonly(CaseData::getMinipetitionlink)

Reply:
Renamed it to miniapplicationlink